evalml/tests/pipeline_tests/regression_pipeline_tests/test_multiseries_regression_pipeline.py

```diff
@@ -356,7 +356,7 @@ def test_time_series_pipeline_get_prediction_intervals(
     )
     mock_transform_return_value = (
         mock_X,
-        pd.DataFrame(np.random.rand(13, 5)),
+        mock_y,
```

Comment:
anyway we can add coverage here?

Reply:
maybe [here](https://github.com/alteryx/evalml/blob/main/evalml/tests/component_tests/decomposer_tests/test_stl_decomposer.py#L151)?